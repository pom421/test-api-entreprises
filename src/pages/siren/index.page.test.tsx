import QueryPage from "@/pages/siren/index.page"
import { render } from "@testing-library/react"

test("render as expected", () => {
  render(<QueryPage />)

  expect(document.body.innerHTML).toMatchSnapshot()
})
