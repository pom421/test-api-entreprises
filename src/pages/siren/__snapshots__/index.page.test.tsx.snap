// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render as expected 1`] = `"<div id=\\"chakra-toast-portal\\"><ul id=\\"chakra-toast-manager-top\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;\\"></ul><ul id=\\"chakra-toast-manager-top-left\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;\\"></ul><ul id=\\"chakra-toast-manager-top-right\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;\\"></ul><ul id=\\"chakra-toast-manager-bottom-left\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;\\"></ul><ul id=\\"chakra-toast-manager-bottom\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;\\"></ul><ul id=\\"chakra-toast-manager-bottom-right\\" style=\\"position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;\\"></ul></div><div><div><div class=\\"css-zf0iqh\\"><main class=\\"css-1rr4qq7\\"><div role=\\"group\\" class=\\"chakra-form-control css-0\\"><label for=\\"siren\\" id=\\"field-1-label\\" class=\\"chakra-form__label css-uvho5g\\">SIREN</label><input type=\\"text\\" placeholder=\\"SIREN sur 9 caractÃ¨res\\" id=\\"siren\\" class=\\"chakra-input css-0\\" value=\\"\\"></div><div class=\\"css-gg4vpm\\"><div class=\\"chakra-stack css-234deb\\"><a disabled=\\"\\" class=\\"chakra-button css-5c2176\\" href=\\"https://annuaire-entreprises.data.gouv.fr/entreprise/\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Annuaire entreprise&nbsp;<svg viewBox=\\"0 0 24 24\\" focusable=\\"false\\" class=\\"chakra-icon css-2y6psj\\"><g fill=\\"none\\" stroke=\\"currentColor\\" stroke-linecap=\\"round\\" stroke-width=\\"2\\"><path d=\\"M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6\\"></path><path d=\\"M15 3h6v6\\"></path><path d=\\"M10 14L21 3\\"></path></g></svg></a><a disabled=\\"\\" class=\\"chakra-button css-5c2176\\" href=\\"https://www.pappers.fr/entreprise/\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Pappers&nbsp;<svg viewBox=\\"0 0 24 24\\" focusable=\\"false\\" class=\\"chakra-icon css-2y6psj\\"><g fill=\\"none\\" stroke=\\"currentColor\\" stroke-linecap=\\"round\\" stroke-width=\\"2\\"><path d=\\"M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6\\"></path><path d=\\"M15 3h6v6\\"></path><path d=\\"M10 14L21 3\\"></path></g></svg></a></div><div class=\\"chakra-stack css-84zodg\\"><p class=\\"chakra-text css-0\\">Exemples de Siren :</p><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">443007778</button><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">345311765</button><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">412653180</button><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">850699323</button><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">771200318</button><button type=\\"button\\" class=\\"chakra-button css-gfrnv\\">418180683</button></div></div><div class=\\"chakra-stack css-17vvmxc\\"></div></main><footer class=\\"css-uewl2b\\"><a href=\\"https://github.com/pom421/test-api-entreprises\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\"><div class=\\"chakra-stack css-14lxv93\\"><div class=\\"css-0\\">Contribuez sur GitHub&nbsp;</div><img alt=\\"GitHub logo\\" src=\\"https://github.githubassets.com/images/modules/site/icons/footer/github-mark.svg\\" class=\\"chakra-image css-1ko1n4n\\"></div></a></footer></div></div></div>"`;
